### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/fbopen-docs#16]
Results can be recreated using [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/)

## Results for https://pages.18f.gov/fbopen:
* __Error:__ The html element should have a lang or xml:lang attribute which describes the language of the document.
 * WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
 * html
 * `<html><head> <meta charset="utf-8...</html>`

* __Error:__ Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
 * WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
 * html > body > div > header > a > img
 * `<img src="/fbopen/assets/fbopen.png">`

* __Error:__ Anchor element found with a valid href attribute, but no link content has been supplied.
 * WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
 * html > body > div > footer > p:nth-child(1) > font > a:nth-child(1)
 * `<a href="http://twitter.com/fbopen"><i class="fa fa-twitter-square"...</a>`

* __Error:__ Anchor element found with a valid href attribute, but no link content has been supplied.
 * WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
 * html > body > div > footer > p:nth-child(1) > font > a:nth-child(2)
 * `<a href="http://github.com/18f/fbopen"><i class="fa fa-github-square">...</a>`

* __Error:__ Anchor element found with a valid href attribute, but no link content has been supplied.
 * WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
 * html > body > div > footer > p:nth-child(1) > font > a:nth-child(3)
 * `<a href="mailto:fbopen@gsa.gov"><i class="fa fa-envelope-square...</a>`

## Summary:
* 5 Errors
